docs/overview.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: default
+parent: ClusterFuzzLite
+title: Overview
+has_children: false
+nav_order: 2
+permalink: /overview/
+---
+# Overview
+
+
+## Introducing fuzzing with libFuzzer and Sanitizers.
+
+This section provides an overview of the fuzzing process and defines common terms. 
+If you are already familiar with the process of fuzzing, feel free to skip to [Build Integration] 
```

Comment:
"process of fuzzing"
isn't as good as saying "libFuzzer and Sanitizers"
People familiar with fuzzing but not libFuzzer and Sanitizers should read this.